### ISSUE TITLE
feature request: build in editor - support for markdown or markdown editor plugin

### ISSUE THREAD

**Issue by aisbergde** [s9y/Serendipity#715]
I found the best way for me is to write articles in markdown. The syntax is easy and clear. But the buttons in the build in s9y editor are not useful for markdown, they generate HTML Tags.

It would be fine if I could switch to a markdown edit modus.  
A good idea would be an option to set markdown as default editor.  
A separate plugin also would be fine. Something simple like this github editor which I currently use to write this feature request. Maybe there are some existing components which could be used in s9y?

What I also like very much: images can be easy added by drag and drop, by copy and paste.

**Comment by onli** [comment]
I agree: It would be nice to support markdown better. Yhe buttons could output markdown when it's appropriate. We could detect that the markdown plugin is loaded and switch the output they produce?

> A separate plugin also would be fine. Something simple like this github editor which I currently use to write this feature request. Maybe there are some existing components which could be used in s9y?

I'd like that. It's a good hybrid between wysiwyg and real markup. https://tiptap.scrumpy.io/markdown-shortcuts looks great to me and like it could be integrated. Not sure how easy it would be. My own blog does not use markdown, so it would be better if someone else drives this...

> What I also like very much: images can be easy added by drag and drop, by copy and paste.

Sadly that's absolutely not fitting to the current concept of the media library. tipdown supports that though, so it could maybe be added externally in a plugin providing that editor.
